helm_remote/Tiltfile

```diff
@@ -44,7 +44,17 @@ def helm_remote(chart, repo_url='', repo_name='', values=[], set=[], namespace='
     if namespace == '':
         namespace = 'default'
     if repo_url != '':
-        local('helm repo add %s %s' % (repo_name, repo_url))  # updates if already added
+        repo_add_command = 'helm repo add %s %s' % (repo_name, repo_url)
+
+        # Add authentification for adding the repository if credentials are provided
+        if version != '':
+            repo_add_command += ' --version %s' % version
```

Comment:
This will add `--version` to both the `helm pull` AND `helm repo add` commands, but it's not a supported flag for `helm repo add`

Reply:
Yeah it is why i was asking for some feedback if needed